extensions/blocks/podcast-player/components/playlist.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { memo } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { STATE_ERROR } from '../constants';
+
+const TrackError = memo( ( { link } ) => (
+	<div className="podcast-player__episode-error">
+		{ __( 'Episode unavailable', 'jetpack' ) }{ ' ' }
+		{ link && (
+			<span>
+				{ '(' }
+				<a href={ link } rel="noopener noreferrer nofollow" target="_blank">
+					{ __( 'Open in a new tab', 'jetpack' ) }
+				</a>
+				{ ')' }
+			</span>
+		) }
+	</div>
+) );
+
+const Track = memo( ( { track, isActive, isError, selectTrack, index } ) => {
+	return (
+		<li
+			className={ classnames( 'podcast-player__episode', {
+				'is-active': isActive,
+			} ) }
+		>
+			<a
+				className="podcast-player__episode-link"
+				href={ track.link }
+				role="button"
+				aria-pressed="false"
+				onClick={ e => {
+					// Prevent handling clicks if a modifier is in use.
+					if ( e.shiftKey || e.metaKey || e.altKey ) {
+						return;
+					}
+
+					// Prevent default behavior (opening a link).
+					e.preventDefault();
+
+					// Select track.
+					selectTrack( index );
+				} }
+				onKeyDown={ e => {
+					// Only handle the Space key.
+					if ( event.key !== ' ' ) {
+						return;
+					}
+
+					// Prevent default behavior (scrolling one page down).
+					e.preventDefault();
+
+					// Select track.
+					selectTrack( index );
+				} }
+			>
+				<span className="podcast-player__episode-status-icon"></span>
```

Comment:
Needs `aria-hidden` if it has no semantic meaning to a screen reader.

Reply:
Would you mind if we addressed this as part of the task adding icons back. In this PR it is an empty element that never gets any content. I can add it to the task

Reply:
Sure! :) As long as it does get addressed. I think it's the 2nd time I've raised in a PR only to be told it will be addressed in a follow up. I understand we've had a lot of PRs floating around so that's ok, but let's make sure we do it. It's low hanging fruit.

Reply:
I'll make sure it does. All follow-ups from this are already reflected within the individual tasks on the board. I'll keep adding more, like this one

Reply:
FYI - we can add a `<title>` to the SVG with a translated string that indicates the state so the screen reader will announce "Playing: [Track title]", as it does give meaning. 
cc @WunderBart

Reply:
@jeryj can you leave the feedback in the icon PR, please? https://github.com/Automattic/jetpack/pull/15101